examples/twittermap/web/public/javascripts/map/countmap.js

```diff
@@ -499,15 +501,15 @@ angular.module('cloudberry.map')
           // get selected geoID for the polygon
           $scope.selectedPlace = layer.feature;
           $scope.selectedGeoID = $scope.selectedPlace.properties.cityID || $scope.selectedPlace.properties.countyID || $scope.selectedPlace.properties.stateID;
-
-          // bind a pop up window
-          $scope.popUp = L.popup({autoPan:false});
-          layer.bindPopup($scope.popUp).openPopup();
-          $scope.popUp.setContent(getPopupContent()).setLatLng([$scope.selectedPlace.properties.popUpLat,$scope.selectedPlace.properties.popUpLog]);
-
-          addPopupEvent();
-          drawLineChart();
-
+          if ($scope.checkIfQueryIsRequested == true) {
+              // bind a pop up window
```

Comment:
Avoid this type of unnecessary format changes that are not related to the logic.